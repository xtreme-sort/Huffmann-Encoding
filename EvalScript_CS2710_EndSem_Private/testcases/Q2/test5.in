DeriveHuffmanTree
10
5 1110
G 1001
K 1010
L 01
W 110
_ 1011
c 1111
e 10000
r 10001
x 00
Quit
