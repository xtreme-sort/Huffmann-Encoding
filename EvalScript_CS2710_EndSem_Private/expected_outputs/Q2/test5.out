((x,L),((((e,r),G),(K,_)),(W,(5,c))));
